playbook/app/pb_kits/playbook/pb_label_value/_label_value.tsx

```diff
@@ -15,7 +15,7 @@ type LabelValueProps = {
   aria?: { [key: string]: string };
   className?: string;
   dark?: boolean;
-  data?: object;
+  data?: Record<string, unknown>;
```

Comment:
There is a `GenericObject` type in my commit [here](https://github.com/powerhome/playbook/pull/3021/commits/e752e034e34b62e0b7e9909445463447f1ed5a9e) if you want to cherry pick and use that instead

Reply:
I was gonna just wait and grab it after. But i'll go ahead do that. Thanks man!